### PR TITLE
reactjs_gmp: Task2

### PR DESCRIPTION
**Pull request by KavvaP17**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "presets": [
+        [
+            "@babel/preset-env"
+        ]
+    ],
+    "plugins": [
+        "@babel/plugin-syntax-dynamic-import",
+        "@babel/plugin-transform-runtime"
+    ]
+}
```
Comment (barysmarukhin) [inline .babelrc]: image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/runtime": "^7.1.5",
+    "@types/html-webpack-plugin": "^3.2.4",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.1",
+    "@types/react-redux": "^7.1.16",
+    "@types/react-router": "^5.1.11",
+    "@types/redux": "^3.6.0",
```
Comment (barysmarukhin) [inline package.json:39]: The types for redux are here, but there is no redux itself. Maybe it makes no sense to have something you don't use yet. Or you can install redux right now. Anyway you will need to have it.

Comment (barysmarukhin) [reply package.json:39]: You will also need 'formik', 'yup', 'react-testing-library', 'lodash', 'jest' later on.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    "babel-polyfill": "^6.26.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^5.0.2",
+    "file-loader": "^6.2.0",
+    "html-loader": "^2.1.0",
+    "html-webpack-plugin": "^5.2.0",
+    "mini-css-extract-plugin": "^1.3.8",
+    "sass": "^1.32.8",
+    "sass-loader": "^11.0.1",
```
Comment (barysmarukhin) [inline package.json]: You will be using sass for styling. It is a good oldschool approach, but you can consider something modern like https://cssinjs.org/react-jss/?v=v10.5.1 if you want.

File: webpack/webpack.common.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+module.exports = {
+    entry: {
+        app: './src/index.tsx'
+    },
+    output: {
+        filename: '[name].js',
+        path: path.resolve(__dirname, 'dist')
+    },
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js']
```
Comment (barysmarukhin) [inline webpack/webpack.common.js]: Once you chose to use typescript maybe there is no need to resolve '.js' extension ?

File: .babelrc (hunk 1 above)
Comment (barysmarukhin) [reply .babelrc]: Please, install and use `eslint` which would help you to resolve unfortunate warnings

File: package.json (hunk 2 above)
Comment (barysmarukhin) [reply package.json:39]: 'react-error-boundary'

Comment (KavvaP17) [reply package.json:39]: Thanks, I will install these packages when I start using it.

File: package.json (hunk 3 above)
Comment (KavvaP17) [reply package.json]: Thanks, removed unnecessary packages and install react-jss package.

File: .babelrc (hunk 1 above)
Comment (KavvaP17) [reply .babelrc]: Thanks, done.

**Pull request merged**